CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 ## Next Version
+### Added
+- Add optional passthrough for tuples.
+- Add optional passthrough for ints, that do not fit into an i64.
```

Comment:
Add `by [@TariqTNO] (https://https://github.com/aviramha/ormsgpack/pull/64)`

Reply:
Done. Thanks for seeing this through.